docs/multimethod_docs.rst

```diff
@@ -0,0 +1,71 @@
+.. currentmodule:: uarray
+
+.. _mmauthordocs:
+
+Documentation for multimethod authors
```

Comment:
This section is hard to understand. To start with the title: this is aimed at library authors wanting to provide a backend system, so other packages can implement a backend. That's not really conveyed by the title or the text below.

Then, it misses a description of what that backend will actually look like and how to implement it. Concretely, if say `scipy.fft` wants to implement a backend system, how does it provide its context manager and multimethods? That's the most important part. Once you have that, then go into how to write a single multimethod.